src/App.js

```diff
@@ -37,8 +37,9 @@ function App() {
   };
 
   useEffect(() => {
-    reGenerate(isKata);
-  }, [isKata]);
+    reGenerate();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
actually if I remembered, I was using this one?

Reply:
Yes, that was my bad, I changed the behavior on that commit. I think it's now reverted?